src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs

```diff
@@ -159,6 +159,7 @@ private async Task<bool> ExecuteAsync(Common.ILogger log)
                 forceEvaluate: RestoreForceEvaluate,
                 hideWarningsAndErrors: HideWarningsAndErrors,
                 restorePC: RestorePackagesConfig,
+                cleanupAssetsForUnsupportedProjects: false,
```

Comment:
was this an accident, perhaps some debugging code that was forgotten to be reverted?   It's not intuitive to me how it's related to this PR's title

Reply:
I missed it when I was reverting previous edits